recipes/xamarin.mac.recipe

```diff
@@ -0,0 +1,29 @@
+# -*- Mode: Python -*- vi:si:et:sw=4:sts=4:ts=4:syntax=python
+import shutil
+import cerbero.utils.messages as m
+from cerbero.utils import shell
+
+class Recipe(recipe.Recipe):
+    name = 'Xamarin.Mac'
+    version = '4.6.1'
+    licenses = [License.LGPL]
+    stype = SourceType.CUSTOM
+    btype = BuildType.CUSTOM
+
+    files_lib = [
+        'lib/libxammac.dylib',
+    ]
+    files_cli = [
+        'lib/mono/4.5/Xamarin.Mac.dll',
+    ]
+
+    def prepare(self):
+        if self.config.platform != Platform.DARWIN:
+            raise InvalidRecipeError()
+
+    def install(self):
+        xamarinmac = "/Library/Frameworks/Xamarin.Mac.framework/Versions/Current/"
```

Comment:
Shouldn't this alert to install Xamarin.Mac.Framework in the prepare method if it doesn't exists?